### DISCUSSION TITLE
"unsupported user role" when trying to change a user to type "user"

### DISCUSSION THREAD

**Discussion by chmeeedalf** [photoprism/photoprism#3734] [Q&A] [answered]
I just registered for PhotoPrism Essentials, but when I tried to change a user role to "user" using the command

photoprism --defaults-yaml= --assets-path=/path/to/assets --storage-path=/path/to/storage --originals-path=/path/to/originals --import-path=/path/to/import users mod --role "user" <username>

I see the following messages:

migrate: running database migrations
config: password required to initialize 'admin' account
unsupported user role

I'm running PhotoPrism on FreeBSD built from the port at https://github.com/huo-ju/photoprism-freebsd-port if that makes any difference.

**Comment by lastzero** [comment, marked as answer]
My apologies! I believe this needs to be communicated more clearly (beyond what's in our [Membership FAQ](https://www.photoprism.app/membership/faq#how-can-i-install-photoprism-plus-without-the-docker-image)) and we should find a better solution for FreeBSD users.

We recently made the roles from our private Plus Extension repository also available to Essentials members when they use the official Docker image we provide. Almost all other Essentials features are included in our [public repository](https://github.com/photoprism/photoprism), except for these roles. As a result, they are not currently available in the latest FreeBSD port, which is built from our public repository only.

Feel free to email us so we can find a solution :)

**Reply by lastzero** [reply]
We have updated the feature table and FAQ on our website to be more clear about this limitation when building from source:
- https://www.photoprism.app/editions#compare
- https://www.photoprism.app/membership/faq#how-can-i-install-photoprism-plus-without-the-docker-image

**Comment by chmeeedalf** [comment]
Hi, thanks for this information.  Could you add a note in the first comparison page for Essentials for this as well?  Just a note in the "Account roles" row, even just a footnote link, that it's (currently?) only available in the docker image?  I see it in the detailed feature comparison now, but it's easy to miss this detail by looking at the first "personal membership" page comparison table.

**Reply by lastzero** [reply]
Sure! We have already added a yellow info icon with link to the feature overview on https://www.photoprism.app/editions#compare and the https://www.photoprism.app/features page. On which other pages/URLs should we add a notice?

**Reply by chmeeedalf** [reply]
Ah, I see that now.  I guess it may have been a cache issue on my side that I missed it before, or something.
